src/tokens/BlessCurseManager.ttslua

```diff
@@ -1,139 +1,101 @@
-        position = { 0.7, 0.1, -0.5 },
-        height = 150,
-        width = 350,
-        scale = { x = 1.75, y = 1.75, z = 1.75 },
-        font_size = 100,
-        font_color = { r = 1, g = 1, b = 1 },
-        color = BLESS_COLOR
-    })
-
-    self.createButton({
-        label = "Return",
-        click_function = "returnBlessToken",
-        function_owner = self,
-        position = { 2.1, 0.1, -0.5 },
-        height = 150,
-        width = 400,
-        scale = { x = 1.75, y = 1.75, z = 1.75 },
-        font_size = 100,
-        font_color = { r = 1, g = 1, b = 1 },
-        color = BLESS_COLOR
-    })
+-- common button parameters
+local buttonParamaters          = {}
+buttonParamaters.function_owner = self
+buttonParamaters.color          = { 0, 0, 0, 0 }
+buttonParamaters.width          = 700
+buttonParamaters.height         = 700
 
-    self.createButton({
-        label = "Add",
-        click_function = "addCurseToken",
-        function_owner = self,
-        position = { -2.3, 0.1, 0.5 },
-        height = 150,
-        width = 300,
-        scale = { x = 1.75, y = 1.75, z = 1.75 },
-        font_size = 100,
-        font_color = { r = 1, g = 1, b = 1 },
-        color = CURSE_COLOR
-    })
+local tokenarranger = getObjectFromGUID("022907")
+local MODE          = { [false] = "Add / Remove", [true] = "Take / Return" }
+local BUTTON_COLOR  = { [false] = { 0.4, 0.4, 0.4 }, [true] = { 0.9, 0.9, 0.9 } }
+local FONT_COLOR    = { [false] = { 1, 1, 1 }, [true] = { 0, 0, 0 } }
+local whitespace    = "     "
 
-    self.createButton({
-        label = "Remove",
-        click_function = "removeCurseToken",
-        function_owner = self,
-        position = { -0.9, 0.1, 0.5 },
-        height = 150,
-        width = 450,
-        scale = { x = 1.75, y = 1.75, z = 1.75 },
-        font_size = 100,
-        font_color = { r = 1, g = 1, b = 1 },
-        color = CURSE_COLOR
-    })
+---------------------------------------------------------
+-- creating buttons and menus + initializing tables
+---------------------------------------------------------
 
-    self.createButton({
-        label = "Take",
-        click_function = "takeCurseToken",
-        function_owner = self,
-        position = { 0.7, 0.1, 0.5 },
-        height = 150,
-        width = 350,
-        scale = { x = 1.75, y = 1.75, z = 1.75 },
-        font_size = 100,
-        font_color = { r = 1, g = 1, b = 1 },
-        color = CURSE_COLOR
-    })
+function onSave() return JSON.encode(TOGGLE) end
 
-    self.createButton({
-        label = "Return",
-        click_function = "returnCurseToken",
-        function_owner = self,
-        position = { 2.1, 0.1, 0.5 },
-        height = 150,
-        width = 400,
-        scale = { x = 1.75, y = 1.75, z = 1.75 },
-        font_size = 100,
-        font_color = { r = 1, g = 1, b = 1 },
-        color = CURSE_COLOR
-    })
+function onLoad(saved_state)
+    if saved_state ~= nil then
+        TOGGLE = JSON.decode(saved_state)
```

Comment:
Declare as with other variables above, and rename to standard case since it's not a constant

Reply:
Reading through the rest of the code, this needs a more descriptive name.  It's through the entire file and too generic to know what it's doing.